app/assets/main/react_components/footprint_result/Payment.jsx

```diff
@@ -5,18 +5,23 @@ import React from 'react';
  * Logic-less component for inputing payment details
  * TODO: add stripe logic, including collection of email and password
  */
-const Payment = ({ commonStrings }) => {
-
+const Payment = ({ commonStrings, selectedMembership }) => {
+    console.log(selectedMembership)
+    console.log(selectedMembership === "free")
     return (
       <div className="text-left">
         <label className="block font-semibold mt-3">{commonStrings.email}</label>
         <input className="input w-full"/>
         <label className="block font-semibold mt-3">{commonStrings.password}</label>
         <input className="input w-full"/>
+        { !(selectedMembership === "free") &&
+        <>
         <label className="block font-semibold mt-3">{commonStrings.credit_or_debit_card}</label>
         <CardElement className="py-3 w-full input mb-1 StripeElement StripeElement--empty"/>
         <i className="fas fa-lock" aria-hidden="true"></i>
         <span className="ml-1">Secured by Stripe</span>
+        </>
+        }
```

Comment:
Samma förslag som tidigare kommentar och indenteringsfix bara!
```suggestion
        { selectedMembership !== "free" &&
        <>
            <label className="block font-semibold mt-3">{commonStrings.credit_or_debit_card}</label>
            <CardElement className="py-3 w-full input mb-1 StripeElement StripeElement--empty"/>
            <i className="fas fa-lock" aria-hidden="true"></i>
            <span className="ml-1">Secured by Stripe</span>
        </>
        }
```